.github/workflows/main.yaml

```diff
@@ -59,17 +61,26 @@ jobs:
           cache-dependency-path: go.sum
       - name: build
         run: make build
+      - name: build previous release
+        run: |
+          git worktree add ../prev v0.1.0
```

Comment:
Is the previous version supposed to always be lagging one release behind, or is `v0.1.0` of special interest for a while now?